core/src/com/unciv/ui/pickerscreens/FoundReligionPickerScreen.kt

```diff
@@ -0,0 +1,103 @@
+package com.unciv.ui.pickerscreens
+
+import com.badlogic.gdx.scenes.scene2d.ui.*
+import com.badlogic.gdx.utils.Align
+import com.unciv.UncivGame
+import com.unciv.logic.GameInfo
+import com.unciv.logic.civilization.CivilizationInfo
+import com.unciv.models.UncivSound
+import com.unciv.models.ruleset.Belief
+import com.unciv.models.translations.tr
+import com.unciv.ui.utils.*
+import kotlin.math.max
+
+class FoundReligionPickerScreen (
+    private val choosingCiv: CivilizationInfo,
+    private val gameInfo: GameInfo
+): PickerScreen() {
+
+    // Roughly follows the layout of the original (although I suck at UI designing, so please improve this)
+    private val topReligionIcons = Table() // Top of the layout, contains icons for religions
+    private val leftChosenBeliefs = Table() // Left middle part, contains buttons to select the types of beliefs to choose
+    private val rightBeliefsToChoose: ScrollPane // Right middle part, contains the beliefs to choose
+    
+    private val middlePanes: SplitPane
+    private val newPanes: SplitPane
+ 
+    private var previouslySelectedIcon: Button? = null
+    private var iconName: String? = null
+    private var religionName: String? = null
+    private var chosenFounderBelief: Belief? = null
+    private var chosenFollowerBelief: Belief? = null
+
+    init {
+        closeButton.isVisible = true
+        setDefaultCloseAction()
+        
+        stage.clear()
+        
+        setupReligionIcons()
+                
+        rightBeliefsToChoose = ScrollPane(Table())
+        
+        descriptionLabel.center(bottomTable)
+        
+        middlePanes = SplitPane(leftChosenBeliefs, rightBeliefsToChoose, false, skin)
+        middlePanes.splitAmount = 0.5f
+        
+        newPanes = SplitPane(topReligionIcons, middlePanes, true, skin)
+        newPanes.splitAmount = 0.15f
```

Comment:
Please don't use SplitPane, we're trying to get rid of them wherever possible

Reply:
What should I use then? Sorry, I suck at UI and basically copy+pasted the PickerScreen() until it worked

Reply:
Table()